cirq-aqt/cirq_aqt/aqt_device.py

```diff
@@ -24,9 +24,14 @@
 """
 
 import json
-from typing import Any, cast, Dict, Optional, Sequence, List, Tuple, Union
+from typing import Any, cast, Dict, Iterable, List, Optional, Sequence, Set, Tuple, Union
+
+import networkx as nx
 import numpy as np
+
 import cirq
+from cirq import circuits, devices, ops, protocols, transformers, value
```

Comment:
We already have `import cirq` above; so we don't need to import the individual modules here. Just use top-level cirq objects using `cirq.<object_name>`.

Reply:
Ack.  I'd rather leave the code as is for the sake of finding the chunks moved from `ion_device` -
```
$ git diff -w --color-moved cc17e2c2 567364b9
```

Also `circuits.something` seems to be a more common reference than `cirq.circuits.something`:
```shell
$ git grep '[^.]circuits[.]\w' '*.py' | grep -cvw import
233
$ git grep '[^.]cirq[.]circuits[.]\w' '*.py' | grep -cvw import
20
```

Reply:
You should be doing `cirq.something` instead of `cirq.circuits.something` here. 

For modules inside `cirq-core`; we need and should import and use only the corresponding submodules (so `circuits.something` instead of `cirq.something`) but for other vendor modules (eg: cirq-aqt, cirq-google, etc.)  we can directly `import cirq` because the statement only imports cirq-core which is now a separate module and hence follow the guidelines that we expect any cirq developer to follow; which is to use top-level cirq objects (i.e. `cirq.something` instead of `circuits.something`). 

This is also the reason why: 
 * Your grep shows more references of using `circuits.something`: We have a lot more code in cirq-core than vendor modules
 * ion_device was earlier using `circuits.something` instead of `cirq.something`: It was part of cirq-core; but now it's not.

Reply:
@pavoljuhas Can you make this change? Do you still have questions / concerns after my explanation?

Reply:
Makes sense, thank you for catching this.
Done in b64aab2e50bb00126377609ea6369cab1e376518.